minerl/Malmo/Minecraft/src/main/java/com/microsoft/Malmo/Client/ClientStateMachine.java

```diff
@@ -1874,9 +1874,15 @@ protected void onMissionStarted()
 
             // Force brightness setting
             Minecraft.getMinecraft().gameSettings.gammaSetting = (float) 2.0;
```

Comment:
A gamma setting of 2 is brighter than playable without editing the options.txt manually (e.g. the highest in game via the brightness slider)

Reply:
May also need to look into fov as it is beyond quake-pro in default minerl

Reply:
good point, but I'd rather get a wider team buy-in before changing these values, since that would affect everyone's experiments. Currently in this PR the behavior (without lowLevelInput flag) should not change.